projects/pastanaga-angular/src/lib/avatar/avatar.component.ts

```diff
@@ -66,14 +66,17 @@ export class AvatarComponent {
     get initials() {
         return this._initials;
     }
+    get userId() {
+        return this._userId;
+    }
 
-    private _userId?: string;
+    private _userId = '';
```

Comment:
if userId is always defined, the background color might be computed in a different way... so we'll get an avatar color based on an empty string